calliar_server/server/static/paper.js

```diff
@@ -23,7 +23,15 @@ var currImageId = 0;
 
 function addRaster(imageName)
 {
-    var raster = new paper.Raster({source: "/static/images/"+imageName})
+    if (imageName==undefined){
+        alert('no more images to draw or something went wrong, going to home page')
+        if (window.location.pathname !='/'){
```

Comment:
@zaidalyafeai 

I think there should be a better way to handle this. Please advice